components/post/postTitle.tsx

```diff
@@ -1,3 +1,5 @@
 export function PostTitle({ title }: { title: string }): JSX.Element {
-  return <h1 className="text-4xl lg:text-6xl font-extrabold">{title}</h1>
+  return <h1 id={title} className="text-4xl lg:text-6xl font-extrabold">
+    <a href={`#${title}`} className="hover:underline">{title}</a>
+  </h1>
```

Comment:
This will only add the anchor to the first h1 on the page, to here:
![Screenshot 2024-04-04 at 14 59 58](https://github.com/kettanaito/kettanaito.com/assets/14984911/660fb8fa-cc25-405f-aff5-35ef19d93719)

I believe your intention was so that _every_ heading in the article has anchor. 

To achieve this, we have to configure MDX used by Next to have some anchor plugin. Those exists, you can look them up if you have a minute.